eng/Versions.props

```diff
@@ -28,6 +29,7 @@
     <MicrosoftDotNetGenAPIPackageVersion>1.0.0-beta.19202.3</MicrosoftDotNetGenAPIPackageVersion>
     <MicrosoftDotNetGenFacadesPackageVersion>1.0.0-beta.19177.11</MicrosoftDotNetGenFacadesPackageVersion>
     <MicrosoftDotNetXUnitExtensionsPackageVersion>2.4.0-beta.19202.3</MicrosoftDotNetXUnitExtensionsPackageVersion>
+    <MicrosoftDotNetXUnitConsoleRunnerPackageVersion>2.4.0-beta.19202.3</MicrosoftDotNetXUnitConsoleRunnerPackageVersion>
```

Comment:
Is this the right version? I see 2.5.1-beta.19202.3 as the version corresponding to this arcade hash.

Reply:
This is just a placeholder and will be auto-updated by maestro.

Reply:
I just updated the version with a manual darc update. Thanks!